### DISCUSSION TITLE
Is there any way to remove the "Powered by flowise" watermark?

### DISCUSSION THREAD

**Discussion by Noe-Briffa** [FlowiseAI/Flowise#626] [Q&A] [answered]
Hello !

I'd like to use Flowise to build a Chatbot for a company.
I'm using embedding to easily implement it on their website.
Of course the "Powered by Flowise" watermark is a bit awkward for the company.
Is there a way to remove it? Or do I have to recode the whole interface myself?

Thanks in advance!

**Comment by Kolaposki** [comment]
I think this can work.
Unminify the js then search for Powered by. Put a style with display none in the span tag.

**Reply by Noe-Briffa** [reply]
Thanks for your reply!

I've already tried that but the problem is on this line:
import Chatbot from "myfile.js"

In fact the js file doesn't contain any Chatbot function...
How is this possible?

**Comment by HenryHengZJ** [comment]
https://docs.flowiseai.com/how-to-use#embed

**Reply by Noe-Briffa** [reply]
Thanks for your reply, but it doesn't seem to me that there's anything on how to remove the watermark.

**Reply by HenryHengZJ** [reply, marked as answer]
see below: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/ae450c69-f0f4-4204-96b9-551430ccb817)

**Reply by Noe-Briffa** [reply]
Thank you very much! I had indeed missed this information!

But I still have a new problem 😇

I copied the GitHub FlowiseChatEmbed, I git clone all this locally and I made some modifications.
I installed yarn and ran yarn build successfully (I think since there doesn't seem to be any error)

I then git pushed all this to my GitHub and tried to load the ChatBot on one of my websites as explained in the tuto (with the correct GitHub😁 address)

The changes appear in my GitHub files, but not on my website...
However, the ChatBot is working and I'm loading it from the right GitHub address...

Has anyone experienced this problem before?

Thanks in advance!

**Reply by Noe-Briffa** [reply]
As far as I'm concerned, it was just time for the CDN to update!
Thank you all, I close the subject!

**Reply by BiasedlyUnbiased** [reply]
I suppose you modified the index.html file? Any idea how to do this if the app is deployed to the cloud like Render for example?

**Comment by remixdance** [comment]
Is there a way to do that with render?

**Reply by Toolfolks** [reply]
Watching....... I need this Flowise 1.5

**Reply by BiasedlyUnbiased** [reply]
Has anyone figured out a way to accomplish this on Render?

**Comment by Max-Falcon** [comment]
> https://docs.flowiseai.com/how-to-use#embed

I dont think this method works anymore! The files and folders described in the instructions are not the same in the lates Flowise. How can we remove the braning in the current verison? Thanks

**Reply by Max-Falcon** [reply]
Nevermind, the instructions is ok! I was looking at the the main repository. This instruction is talking about flowiesembed repo which is at https://github.com/FlowiseAI/FlowiseChatEmbed.

**Comment by Max-Falcon** [comment]
How is brand removing done if I already deployed the main repo on AWS?

**Comment by Toolfolks** [comment]
I found this worked on my local installation.
C:\Users\User\node_modules\flowise-ui\build\static\js

file 690.5317b72b.chunk.js
Search for Powered By

replace with your details

**Reply by mejaatg** [reply]
you saved us!!!!

**Reply by toi500** [reply]
with the new version this does not work anymore. Did u find a fix?

**Reply by Daniel-DDV** [reply]
Did you try this: https://github.com/FlowiseAI/Flowise/discussions/626#discussioncomment-6582289

**Reply by toi500** [reply]
the link does not work https://docs.flowiseai.com/how-to-use#embed. 

Still working with the version 1.6?

**Reply by Daniel-DDV** [reply]
check this out and be sure to follow 100%: https://youtu.be/4MSxi6-RYCI?si=1Dm__4Ht7sglPUBO

**Reply by toi500** [reply]
thanks, man. I appreciate ur time to help.

**Reply by Yasir-Mukthar** [reply]
node_modules\flowise-ui\build\static\js

i don't have this ( static\js )  folder 

C:\Users\yasir\AppData\Roaming\npm\node_modules\flowise\node_modules\flowise-ui\build

how can i remove this powered by flowise

**Reply by toi500** [reply]
The only way to remove it is by:

1. Setting up the color #fff on it

or

2. Following this guide https://youtu.be/4MSxi6-RYCI?si=V6aLYeo-6hkwSp7W

**Reply by BiasedlyUnbiased** [reply]
The tutorial is for a project that is built locally, but how about if it's already deployed to the cloud like Render?

**Comment by BiasedlyUnbiased** [comment]
Is there a way to implement this if the app is deployed to Render?

**Reply by toi500** [reply]
The tutorial works for localhost, Render, or any other provider you want to. What you are doing there is to generate a new web.js (chatbot) file without the `powered by flowise` tag and this new web.js can be hosted anywhere.

**Reply by BiasedlyUnbiased** [reply]
Scratch my last comment...So I did manage to figure out that the embed link should be modified which, technically, should reflect the changes. Unfortunately, the branding persists. Despite following all the steps, including the @latest add. I also found the EmbedChat.jsx which contains the poweredby text color. Changed it to transparent and boom, nothing. So I changed it to white and now I'm waiting to see the results. Anything that I'm missing that you can note would be appreciated.

**Reply by toi500** [reply]
If you did it right and it is not showing your changes is 99% because you used the same file name web.js. The old one is still cached at jsdelivery.com and you need to purge it manually.

1. You can purge the chache here https://www.jsdelivr.com/tools/purge

2. Just change the name of the new web.js that you have generated, something like web-01.js. It must shows your changes now since that file path is now a new one, never cached before.

If that does not work, try this:

```JS

<script type="module">
  import Chatbot from "https://cdn.jsdelivr.net/npm/flowise-embed/dist/web.js";
  Chatbot.init({
    chatflowid: "YOUR_ACTUAL_CHATFLOW_ID", // Replace with your Flowise chatbot ID
    apiHost: "YOUR_ACTUAL_API_HOST",       // Replace with your Flowise API endpoint
    chatWindow: {
      poweredByTextColor: "#fff",
    },
  });
</script>

```

Note, you can change the color of the tag in the embbeding:

```JS
chatWindow: {
      poweredByTextColor: "#fff",
    },
```

**Reply by BiasedlyUnbiased** [reply]
That was helpful, thank you. I tried everything yet the text persisted. It's like it was teasing me hahaha. The funny part is that all the changes are committed and synced and the application runs fine, except, well, the darn text. Anyway, I found a way to generate a full HTML embedding by Render, from which I was able to change the text color to white. Obviously, this isn't a perfect solution as you can still hover on the invisible text and click on the Flowise link, and you also won't be able to add your own text, but it's the best working solution I found so far. I may decide to go another round later, so if you can think of any additional tips please share. Again, thanks for the help.

**Reply by toi500** [reply]
@BiasedlyUnbiased 

I made it for you. I removed the brand from the source code. It is zipped because GH does not allow to upload JS files on comments.

1. Unzip
2. Host the web-nobrand.js file in your own repository
3. Go to this URL to get the file always acccesible (avoiding loading delays) https://www.jsdelivr.com/github
4. Get the file path on jsdelivery and insert it in the flowise script

File:
[web-nobrand.zip](https://github.com/FlowiseAI/Flowise/files/15181049/web-nobrand.zip)

<img width="345" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/138339291/61c246c6-5b0d-4205-a2d3-b2c96592d7fd">

**Reply by IceQboid** [reply]
@BiasedlyUnbiased @toi500 

Another quick very odd fix I found was to wrap the initialization in a function, after updating the forked repo as shown in the YouTube tutorial:

<script type="module">
    import Chatbot from "https://cdn.jsdelivr.net/gh/username/custom-chatembed-repo@latest/dist/web.js";

    function startMyBot() {
      Chatbot.init({
        chatflowid: "CHATFLOW_ID",  
        apiHost: "API_HOST",           
        
      });
    }
    startMyBot();
  </script>
